data_structures/src/superblock.rs

```diff
@@ -516,17 +516,18 @@ pub fn calculate_superblock_signing_committee(
         let index_hash = Hash::from(calculate_sha256(&current_superblock_index.to_be_bytes()));
         let first_byte_sb_hash = *superblock_hash.as_ref().get(0).unwrap();
         let first_byte_index_hash = *index_hash.as_ref().get(0).unwrap();
```

Comment:
you can move those lines into else branch to avoid doing a hash when it is not needed

Reply:
Done!